source/support/zendesk/index.html.md.erb

```diff
@@ -262,29 +262,29 @@ You'll be given 2 options: To subscribe to email updates or to create an account
 
 Thank you for your message and sorry to hear you're having trouble with your account.
 
-You've reached the GOV.UK Account team - we are the team responsible for maintaining the GOV.UK Account. The GOV.UK Account is the one that you can currently use to save your answers from the Brexit checker - to help you work out what you need to do differently now that the UK has left the EU.
+You've reached the GOV.UK Account team - we are the team responsible for maintaining the GOV.UK Account. The GOV.UK Account is the one that you can currently use to save your answers from the [Brexit checker](https://www.gov.uk/transition-check/questions). The Brexit checker helps you work out what you need to do differently now that the UK has left the EU.
 
-Is this the account that you would like to delete? If so I can help. I have included detailed instructions for how you can delete your account below.
+Is this the account that you would like to delete? If so I can help. I have included detailed instructions for how you can delete your account.
 
-But if you were thinking of a different government account (for example Government Gateway, Universal Credit, Personal Tax Account, etc), please can you let me know what service you were trying to use or what you were trying to do? Or if it's easier, you can share the link of the account you were trying to access with me. I will then try to help direct you to the right place.
+But if you were thinking of a different government account (for example Government Gateway, Universal Credit, Personal Tax Account), please can you let me know what service you were trying to use or what you were trying to do? Or if it's easier, you can share the link of the account you were trying to access with me. I will then try to help direct you to the right place.
 
 **How to delete your GOV.UK account**
 
 Step 1: To delete your account, you first need to sign in to your account here: https://www.account.publishing.service.gov.uk/sign-in
 
-If you cannot sign in because you're having password issues, you can simply reset your password by following the instructions here: https://www.account.publishing.service.gov.uk/account/password/reset
+If you cannot sign in because you're having password issues, you can reset your password by following the instructions here: https://www.account.publishing.service.gov.uk/account/password/reset
 
 Once you've reset your password, sign in.
 
 Step 2: Once you've signed in, you need to go to the 'Manage your account' section of your account.
 
-Step 3: Scroll to the bottom and select the 'Delete this GOV.UK account' link.
+Step 3: Scroll to the bottom and select the **Delete this GOV.UK account** link.
```

Comment:
Also yeah won't be able to get that markdown to compile.

We could put the entire answer in a code/quote block perhaps?

```suggestion
Step 3: Scroll to the bottom and select the 'Delete this GOV.UK account' link.
```